templates/zerver/help/view-your-mentions.md

```diff
@@ -1,37 +1,45 @@
 # View your mentions
 
-Zulip keeps track of messages where you have been mentioned and displays the
-number of all your unread @-mentions to allow you to use this feature as an
-inbox of messages you'd like to come back to.
+You can [mention a user or a group](/help/mention-a-user-or-group) to call their
+attention to a message. To make such messages easy to find, Zulip lets you view
+the messages where you were mentioned from a dedicated tab.
 
-## Access your mentions
+!!! warn ""
+
+    Because [silent mentions](/help/mention-a-user-or-group#silently-mention-a-user)
+    are designed not to attract attention, they are excluded from the **Mentions** tab.
+
+## View your mentions
 
 {start_tabs}
 
 {tab|desktop-web}
 
-1. Click **@ Mentions** in the left sidebar.
+1. Click **Mentions** in the left sidebar.
 
```

Comment:
I removed the `@` here to be consistent with how we refer to other tabs in the left sidebar.